templates/editcontent/fields/image.twig

```diff
@@ -43,12 +43,12 @@
 
         </div>
         <div class="col-4" id="{{ id }}-uploader" style="padding: 0; min-height: 180px;">
-            <file-pond name="filepond[]" ref="pond" accepted-file-types="image/jpeg, image/png" server="/async/upload"></file-pond>
+            {# <file-pond name="filepond[]" ref="pond" accepted-file-types="image/jpeg, image/png" server="/async/upload"></file-pond> #}
```

Comment:
We can make these work another way, right? I assume we also need to pull them in, instead of CDN'ing them.

Reply:
Yes I will make a component for it then import it inside that